packages/react-cards/src/components/Card/examples/Card.Vertical.Example.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react';
+import { ActionButton, Button, IButtonStyles, IButtonTokens, Persona } from '@uifabric/experiments';
+import { Card, ICardTokens, ICardItemTokens, ICardSectionStyles, ICardSectionTokens } from '@uifabric/react-cards';
+import { FontWeights } from '@uifabric/styling';
+import { Icon, IIconStyles, Image, Stack, IStackTokens, Text, ITextStyles } from 'office-ui-fabric-react';
+
+const alertClicked = (): void => {
+  alert('Clicked');
+};
+
+export class CardVerticalExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const siteTextStyles: ITextStyles = {
+      root: {
+        color: '#025F52',
+        fontWeight: FontWeights.semibold
+      }
+    };
+    const descriptionTextStyles: ITextStyles = {
+      root: {
+        color: '#333333',
+        fontWeight: FontWeights.semibold
+      }
+    };
+    const helpfulTextStyles: ITextStyles = {
+      root: {
+        color: '#333333',
+        fontWeight: FontWeights.regular
+      }
+    };
+    const iconStyles: IIconStyles = {
+      root: {
+        color: '#0078D4',
+        fontSize: 16,
+        fontWeight: FontWeights.regular
+      }
+    };
+    const footerCardSectionStyles: ICardSectionStyles = {
+      root: {
+        borderTop: '1px solid #F3F2F1'
+      }
+    };
+    const backgroundImageCardSectionStyles: ICardSectionStyles = {
+      root: {
+        backgroundImage: 'url(https://placehold.it/256x144)',
+        backgroundPosition: 'center center',
+        backgroundSize: 'cover',
+        height: 144
+      }
+    };
+    const dateTextStyles: ITextStyles = {
+      root: {
+        color: '#505050',
+        fontWeight: 600 as 600
```

Comment:
`as number`? This is odd, is this a fontWeight typing issue that needs to be resolved?

Reply:
It was when I wasn't specifying typings for these styles/tokens, but it can now be safely removed. Nice catch!